pages/common/naabu.md

```diff
@@ -0,0 +1,29 @@
+# naabu
+
+> A fast port scanner written in go with a focus on reliability and simplicity.
+> Some features only activate when naabu is run with root privileges such as SYN scan.
```

Comment:
```suggestion
> A fast port scanner written in Go with a focus on reliability and simplicity.
> Note: Some features are only activated when `naabu` is run with root privileges such as SYN scan.
```
I have a few suggestions here:

1. We can capitalize Go.
2. We can convert the second line to a note and enclose the command name inside backticks.